NuKeeper/Engine/EngineReport.cs

```diff
@@ -8,27 +7,39 @@ namespace NuKeeper.Engine
 {
     public static class EngineReport
     {
-        public static string PackagesFound(List<PackageInProject> packages)
+        public static string PackagesFoundSummary(List<PackageInProject> packages)
         {
             var projectPathCount = packages
                 .Select(p => p.Path)
                 .Distinct()
                 .Count();
 
             var packageIds = packages
+                .Select(p => p.Id)
+                .Distinct();
+
+            return $"Found {packages.Count} packages in use, {packageIds.Count()} distinct, in {projectPathCount} projects.";
+        }
+
+        public static string PackagesFoundDetails(List<PackageInProject> packages)
```

Comment:
something about this is making me think we should design it more like a proper UI layer rather than just piggybacking on the logger.